### PR TITLE
Feat: remove last seen refactor flag

### PR DESCRIPTION
**Pull request by FredrikOseberg**
What it says on the box

### CODE REVIEW

File: src/test/e2e/api/admin/feature-archive.e2e.test.ts [hunk 1]
```diff
@@ -77,14 +77,13 @@ test('returns three archived toggles', async () => {
 });
 
 test('returns three archived toggles with archivedAt', async () => {
-    expect.assertions(3);
```
Comment (FredrikOseberg) [inline src/test/e2e/api/admin/feature-archive.e2e.test.ts:80]: Why did this have to change?

File: src/lib/services/scheduler-service.test.ts [hunk 2]
```diff
@@ -0,0 +1,122 @@
+import { LogProvider } from '../logger';
```
Comment (FredrikOseberg) [inline src/lib/services/scheduler-service.test.ts:1]: Why is this file new?

File: src/test/e2e/api/admin/feature-archive.e2e.test.ts (hunk 1 above)
Comment (andreas-unleash) [reply src/test/e2e/api/admin/feature-archive.e2e.test.ts:80]: no `archive`, just `archivedAt`

File: src/lib/services/scheduler-service.test.ts (hunk 2 above)
Comment (andreas-unleash) [reply src/lib/services/scheduler-service.test.ts:1]: Not sure, I rebased the branch from main

File: src/test/e2e/api/admin/feature-archive.e2e.test.ts (hunk 1 above)
Comment (andreas-unleash) [reply src/test/e2e/api/admin/feature-archive.e2e.test.ts:80]: I know, that's why I said I had to fix unrelated tests around this, the scheduled-service and maintenance mode. Strange

**Review by andreas-unleash — APPROVED** [review]

**Pull request merged**